ports/stm32/machine_i2s.c

```diff
@@ -790,7 +811,9 @@ STATIC void machine_i2s_init_helper(machine_i2s_obj_t *self, size_t n_pos_args,
     init->MCLKOutput = I2S_MCLKOUTPUT_DISABLE;
     init->AudioFreq = args[ARG_rate].u_int;
     init->CPOL = I2S_CPOL_LOW;
+    #if !defined(STM32H7)
```

Comment:
similar comment as above about processor macro vs feature macro

Reply:
The H7 HAL just doesn't have this entry in the Init struct.  So somehow need to deal with a HAL that is not 100% cross-MCU compatible.